demo/src/app/components/+modal/demos/service-options/custom-css-class/custom-css-class.html

```diff
@@ -0,0 +1,15 @@
+<button type="button" class="btn btn-primary" (click)="openModal(template)">Open modal</button>
```

Comment:
why do we need a regular modal in the demo that shows a modal with css class?

Reply:
yeap, don't needed